modules/admin_manual/pages/configuration/server/security/oauth2.adoc

```diff
@@ -50,6 +56,48 @@ The app aims to:
 . Connect ownCloud clients (both desktop and mobile) in a standardized and secure way.
 . Make 3rd party software integrations easier by providing an unified authorization interface.
 
+=== Requirements
+
+To use the OAuth2 app, your ownCloud installation will need to meet the following dependencies:
+
+* *Apache:* If you are hosting your ownCloud installation using the Apache web server, then {mod_rewrite-url}[mod_rewrite] and {mod_headers-url}[mod_headers] modules must be installed and enabled.
+. *Redis:* You will need to have a Redis server available, ideally the latest, stable, version
+. *PHP:* You PHP installation must have {predis-extension-url}[the phpredis extension] (>= 4.2) installed and enabled. 
+In addition, it must be configured to xref:configuration/server/caching_configuration.adoc#redis[use Redis as a memory cache].
```

Comment:
"PHP needs to be configured to store the sessions in redis" (or something along those lines) seems more accurate.

In addition, I'm not sure if using redis is a requirement, or it's highly recommended.

Reply:
> In addition, I'm not sure if using redis is a requirement, or it's highly recommended.

After looking through the code, I guess it's only a recommendation, as I don't see a hard requirement for it.

Reply:
> "PHP needs to be configured to store the sessions in redis" (or something along those lines) seems more accurate.

Fine by me.